dts/bindings/riscv/riscv,cpus.yaml

```diff
@@ -24,8 +24,3 @@ properties:
       - rv32imafcb
       - rv64imac
       - rv64imafdc
-
-  riscv,pmpregions:
-    description: Physical Memory Protection (PMP)
-    required: true
-    type: int
```

Comment:
Why are you removing this? This removal does not seem to be related to this PR.

Reply:
The property was marked as required but not defined or used anywhere. Is changing to "required: false" preferred instead of removing?

Reply:
A separate commit explaining the removal would be better.

Reply:
Done.